recipes/ffmpeg/all/conanfile.py

```diff
@@ -244,6 +253,10 @@ def _dependencies(self):
     def _version_supports_libsvtav1(self):
         return Version(self.version) >= "5.1.0"
 
+    @property
+    def _version_supports_harfbuzz(self):
```

Comment:
```suggestion
    def _version_supports_harfbuzz(self):
        # https://github.com/FFmpeg/FFmpeg/compare/n6.0.1...n6.1#diff-90d08e583c4c9c6f391b2ae90f819f600a6326928ea9512c9e0c6d98e9f29ac2R235
```

Information verifying the feature support.

Reply:
Added comment justifying harfbuzz version support.